zstd-safe/zstd-sys/build.rs

```diff
@@ -92,8 +92,10 @@ fn compile_zstd() {
         #[cfg(feature = "legacy")]
         "zstd/lib/legacy",
     ] {
-        for entry in fs::read_dir(dir).unwrap() {
-            let path = entry.unwrap().path();
+        let mut entries: Vec<_> = fs::read_dir(dir).unwrap().map(|r| r.unwrap()).collect();
+        entries.sort_by_key(|k| k.path());
```

Comment:
Note: `DirEntry::path()` actually creates a new `PathBuf` everytime.

Since we only care about these paths, it may be easier to collect the Paths directly.